addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -5615,6 +5597,69 @@ public MRESReturn Client_GetMaxHealth(int iClient, Handle hReturn)
 	return MRES_Ignored;
 }
 
+public MRESReturn Client_OnGiveNamedItem(int iClient, Handle hReturn, Handle hParams)
+{
+	static int iBlockedIndexes[] = {
+		57,  //Razorback
+		231, //Darwin's Danger Shield
+		642, //Cozy Camper
+		133, //Gunboats
+		444, //Mantreads
+		405, //Ali Baba's Wee Booties
+		608  //The Bootlegger
+	};
+	
+	char sClassname[256];
+	DHookGetParamString(hParams, 1, sClassname, sizeof(sClassname));
+	
+	int iDefIndex = DHookGetParamObjectPtrVar(hParams, 3, 4, ObjectValueType_Int) & 0xFFFF;
```

Comment:
I wonder if it worth to have 4 offset to gamedata keys. Incase valve ever change this, but doubt that would ever happen anyway.
Also `iIndex` instead of `iDefIndex`

Reply:
Big doubt that valve will ever touch `CEconItemView` so I'll keep it as is